src/View/Canvas.coffee

```diff
@@ -32,8 +32,8 @@ R.create "Canvas",
         className: R.cx {
           LayoutMode: true
           FullScreen: layout.fullScreen
-          "icon-fullscreen": !layout.fullScreen 
-          "icon-edit": layout.fullScreen 
+          "icon-fullscreen": !layout.fullScreen
+          "icon-edit": layout.fullScreen
```

Comment:
This is from my editor automatically removing trailing whitespace, which is generally a nice thing to do. (We may want to add linting to enforce this?)